### PR TITLE
Added the mini_elixivy keyboard

### PR DESCRIPTION
**Pull request by minibois**
Added a new keyboard, mini eLiXiVy (mini_elixivy). Added a default, ANSI and ISO keymap for it.

## Description
Added the keyboard and keymaps for a PCB I've made

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mini_elixivy/mini_elixivy.h [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * |10  |11 |12 |13 |14 |15 |16 |17 |18 |19 |1a |1b |1c |  1d|1e |
+ * |-------------------------------------------------------------|
+ * |20   |21 |22 |23 |24 |25 |26 |27 |28 |29 |2a |2b |2c    ||2e |
+ * |-------------------------------------------------------------|
+ * |30     ||32 |33 |34 |35 |36 |37 |38 |39 |3a |3b      |3b |3c |
+ * |-------------------------------------------------------------|
+ * |40  |41  |42  |           46         |49 |4a |4b |4c |4d |4e |
+ * `-------------------------------------------------------------'
+*/
+#define ansi( \
```
Comment (zvecr) [inline keyboards/mini_elixivy/mini_elixivy.h:37]:
```suggestion
#define LAYOUT_ansi( \
```

File: keyboards/mini_elixivy/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Prts|
+     * |----------------------------------------------------------------|
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Enter  |Del |
+     * |----------------------------------------------------------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|‘ ~ |
+     * |----------------------------------------------------------------|
+     * |Ctrl|Win |Alt |        Space          |Alt|FN |Ctrl|Lef|Dow|Rig |
+     * `----------------------------------------------------------------'
+     */
+    [_BASE] = ansi(
```
Comment (zvecr) [inline keyboards/mini_elixivy/keymaps/default/keymap.c:38]:
```suggestion
    [_BASE] = LAYOUT_ansi(
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/mini_elixivy/mini_elixivy.h (hunk 1 above)
Comment (minibois) [reply keyboards/mini_elixivy/mini_elixivy.h:37]: Hey @zvecr ,
Yesterday when I made this PR one of the QMK Discord mods told me (through Discord) to change my layout names from "layout_ANSI" to "ansi" (and so on for the ISO layout), telling me "we don't accept uppercase".

I just want to be sure "LAYOUT_ansi" (and so on for the ISO layout) is the correct naming, as opposed to what I have in the PR now.

Just asking for clarification, on if upper-case is not accepted at all in the layout names, or if "LAYOUT_ansi" is the convention for QMK

Cheers,
minibois.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mini_elixivy/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# mini_elixivy
+
+![mini_eLiXiVy](https://imgur.com/WSCrhOq.png)
```
[the remark this answers is not in the document]
Comment (minibois) [reply keyboards/mini_elixivy/readme.md]: Hi,
I changed the image to a smaller .jpg image, I assume the suggested change here was a lower resolution and no transparency?

File: keyboards/mini_elixivy/mini_elixivy.h [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+ * |10  |11 |12 |13 |14 |15 |16 |17 |18 |19 |1a |1b |1c |  1d|1e |
+ * |-------------------------------------------------------------|
+ * |20   |21 |22 |23 |24 |25 |26 |27 |28 |29 |2a |2b |2c    ||2e |
+ * |-------------------------------------------------------------|
+ * |30     ||32 |33 |34 |35 |36 |37 |38 |39 |3a |3b      |3b |3c |
+ * |-------------------------------------------------------------|
+ * |40  |41  |42  |           46         |49 |4a |4b |4c |4d |4e |
+ * `-------------------------------------------------------------'
+*/
+#define LAYOUT_ansi( \
```
Comment (drashna) [inline keyboards/mini_elixivy/mini_elixivy.h:37]:
```suggestion
#define LAYOUT_65_ansi( \
```

File: keyboards/mini_elixivy/mini_elixivy.h [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+ * |10  |11 |12 |13 |14 |15 |16 |17 |18 |19 |1a |1b |1c |    |1e |
+ * |-----------------------------------------------------| 1d|---|
+ * |20   |21 |22 |23 |24 |25 |26 |27 |28 |29 |2a |2b |2c |   |2e |
+ * |-------------------------------------------------------------|
+ * |30  |31 |32 |33 |34 |35 |36 |37 |38 |39 |3a |3b      |3b |3c |
+ * |-------------------------------------------------------------|
+ * |40  |41  |42  |           46         |49 |4a |4b |4c |4d |4e |
+ * `-------------------------------------------------------------'
+*/
+#define LAYOUT_iso( \
```
Comment (drashna) [inline keyboards/mini_elixivy/mini_elixivy.h:65]:
```suggestion
#define LAYOUT_65_iso( \
```

File: keyboards/mini_elixivy/rules.mk [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes
```
Comment (drashna) [inline keyboards/mini_elixivy/rules.mk:23]:
```suggestion
ENCODER_ENABLE = yes

LAYOUTS = 65_ansi 65_iso
```

File: keyboards/mini_elixivy/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+/* Rotary Encoder's function (currently volume up/down) */
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (clockwise) {
+        tap_code(KC_AUDIO_VOL_UP);
+    } else {
+        tap_code(KC_AUDIO_VOL_DOWN);
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/default/keymap.c:76]:
```suggestion
```

File: keyboards/mini_elixivy/mini_elixivy.c [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "mini_elixivy.h"
```
Comment (drashna) [inline keyboards/mini_elixivy/mini_elixivy.c:17]:
```suggestion
#include "mini_elixivy.h"

/* Rotary Encoder's function (currently volume up/down) */
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (clockwise) {
        tap_code(KC_VOLU);
    } else {
        tap_code(KC_VOLD);
    }
    return true;
}

```

File: keyboards/mini_elixivy/keymaps/iso/keymap.c [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+/* Rotary Encoder's function (currently volume up/down) */
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (clockwise) {
+        tap_code(KC_AUDIO_VOL_UP);
+    } else {
+        tap_code(KC_AUDIO_VOL_DOWN);
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/iso/keymap.c:76]:
```suggestion
```

File: keyboards/mini_elixivy/keymaps/ansi/keymap.c [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+/* Rotary Encoder's function (currently volume up/down) */
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (clockwise) {
+        tap_code(KC_AUDIO_VOL_UP);
+    } else {
+        tap_code(KC_AUDIO_VOL_DOWN);
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/ansi/keymap.c:76]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]
Looks like this fits the 65% community layouts, and should support those, if so.

File: keyboards/mini_elixivy/readme.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (fauxpark) [reply keyboards/mini_elixivy/readme.md]: Yep. Didn't know the Imgur thumbnails weren't transparent.

File: keyboards/mini_elixivy/info.json [hunk 11]
```diff
@@ -0,0 +1,27 @@
+{
+    "keyboard_name": "mini_elixivy",
+    "url": "https://github.com/minibois/elixivy",
+    "maintainer": "minibois",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_ansi": {
```
Comment (drashna) [inline keyboards/mini_elixivy/info.json:8]:
```suggestion
        "LAYOUT_65_ansi": {
```

File: keyboards/mini_elixivy/info.json [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+                {"label":"Shift", "x":0, "y":3, "w":2.25},{"label":"Z", "x":2.25, "y":3},{"label":"X", "x":3.25, "y":3},{"label":"C", "x":4.25, "y":3},{"label":"V", "x":5.25, "y":3},{"label":"B", "x":6.25, "y":3},{"label":"N", "x":7.25, "y":3},{"label":"M", "x":8.25, "y":3},{"label":",", "x":9.25, "y":3},{"label":".", "x":10.25, "y":3},{"label":"/", "x":11.25, "y":3},{"label":"Shift", "x":12.25, "y":3, "w":1.75},{"label":"Up", "x":14, "y":3},{"label":"F5", "x":15, "y":3},
+                {"label":"Ctrl", "x":0, "y":4, "w":1.25},{"label":"Win", "x":1.25, "y":4, "w":1.25},{"label":"Alt", "x":2.50, "y":4, "w":1.25},{"label":"", "x":3.75, "y":4, "w":6.25},{"label":"Alt", "x":10, "y":4},{"label":"Fn", "x":11, "y":4},{"label":"Ctrl", "x":12, "y":4},{"label":"Left", "x":13, "y":4},{"label":"Down", "x":14, "y":4},{"label":"Right", "x":15, "y":4}
+            ]
+            },
+        "LAYOUT_iso": {
```
Comment (drashna) [inline keyboards/mini_elixivy/info.json:17]:
```suggestion
        "LAYOUT_65_iso": {
```

File: keyboards/mini_elixivy/keymaps/ansi/keymap.c [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Prts|
+     * |----------------------------------------------------------------|
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Enter  |Del |
+     * |----------------------------------------------------------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|‘ ~ |
+     * |----------------------------------------------------------------|
+     * |Ctrl|Win |Alt |        Space          |Alt|FN |Ctrl|Lef|Dow|Rig |
+     * `----------------------------------------------------------------'
+     */
+    [_BASE] = LAYOUT_ansi(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/ansi/keymap.c:38]:
```suggestion
    [_BASE] = LAYOUT_65_ansi(
```

File: keyboards/mini_elixivy/keymaps/ansi/keymap.c [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+     * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |PgUp|
+     * |----------------------------------------------------------------|
+     * |      |   |   |   |   |   |   |   |   |   |   |   |        |PgDn|
+     * |----------------------------------------------------------------|
+     * |        |   |   |   |   |   |   |   |   |   |   |      |   |Ins |
+     * |----------------------------------------------------------------|
+     * |    |    |    |                       |   |   |    |   |   |    |
+     * `----------------------------------------------------------------'
+     */
+    [_FN] = LAYOUT_ansi(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/ansi/keymap.c:59]:
```suggestion
    [_FN] = LAYOUT_65_ansi(
```

File: keyboards/mini_elixivy/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Prts|
+     * |----------------------------------------------------------------|
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Enter  |Del |
+     * |----------------------------------------------------------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|‘ ~ |
+     * |----------------------------------------------------------------|
+     * |Ctrl|Win |Alt |        Space          |Alt|FN |Ctrl|Lef|Dow|Rig |
+     * `----------------------------------------------------------------'
+     */
+    [_BASE] = LAYOUT_ansi(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/default/keymap.c:38]:
```suggestion
    [_BASE] = LAYOUT_65_ansi(
```

File: keyboards/mini_elixivy/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+     * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |    |
+     * |----------------------------------------------------------------|
+     * |      |   |   |   |   |   |   |   |   |   |   |   |        |    |
+     * |----------------------------------------------------------------|
+     * |        |   |   |   |   |   |   |   |   |   |   |      |   |    |
+     * |----------------------------------------------------------------|
+     * |    |    |    |                       |   |   |    |   |   |    |
+     * `----------------------------------------------------------------'
+     */
+    [_FN] = LAYOUT_ansi(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/default/keymap.c:59]:
```suggestion
    [_FN] = LAYOUT_65_ansi(
```

File: keyboards/mini_elixivy/keymaps/iso/keymap.c [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|     |Prts|
+     * |------------------------------------------------------Enter-----|
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '| \|    |Del |
+     * |----------------------------------------------------------------|
+     * |Shift| \|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|‘ ~ |
+     * |----------------------------------------------------------------|
+     * |Ctrl|Win |Alt |        Space          |Alt|FN |Ctrl|Lef|Dow|Rig |
+     * `----------------------------------------------------------------'
+     */
+    [_BASE] = LAYOUT_iso(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/iso/keymap.c:38]:
```suggestion
    [_BASE] = LAYOUT_65_iso(
```

File: keyboards/mini_elixivy/keymaps/iso/keymap.c [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+     * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |PgUp|
+     * |----------------------------------------------------------------|
+     * |      |   |   |   |   |   |   |   |   |   |   |   |        |PgDn|
+     * |----------------------------------------------------------------|
+     * |        |   |   |   |   |   |   |   |   |   |   |      |   |Ins |
+     * |----------------------------------------------------------------|
+     * |    |    |    |                       |   |   |    |   |   |    |
+     * `----------------------------------------------------------------'
+     */
+    [_FN] = LAYOUT_iso(
```
Comment (drashna) [inline keyboards/mini_elixivy/keymaps/iso/keymap.c:59]:
```suggestion
    [_FN] = LAYOUT_65_iso(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**